### PR TITLE
Support override of Mayhem project name

### PR DESCRIPTION
**Pull request by swarkentin**
* Added a new input, `project` which allows action callers to override the `--project` argument that is passed to the `mayhem` CLI. If not set, the existing behavior remains unchanged -- falling back to the GitHub repository name for the project.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -9,35 +9,38 @@ inputs:
     description: URL pointing to your Mayhem instance
     required: false
   owner:
-    description: sets the owner of the project (organization or user).
+    description: Sets the owner of the project (workspace or account name).
     required: false
     default: ${{ github.repository_owner }}
+  project:
```
Comment (vlussenburg) [inline action.yml:15]: Could you used default instead of doing it programmatically in the JS?

Comment (vlussenburg) [reply action.yml:15]: > Defaults to the name of your repository if not defined.

`default: ${{github.event.repository.name}}`

Comment (vlussenburg) [reply action.yml:15]: May not work in all scenarios though...
https://stackoverflow.com/questions/62803531/repository-name-as-a-github-action-environment-variable

Comment (vlussenburg) [reply action.yml:15]:
```
const repo = process.env["GITHUB_REPOSITORY"];
    if (repo === undefined) {
      throw Error(
        "Missing GITHUB_REPOSITORY environment variable. " +
          "Are you not running this in a Github Action environment?"
       );
     }
```

I guess if we had this in the JS, it probably wasn't possible with defaults in the action.yml

**Review by vlussenburg — APPROVED** [review]

File: action.yml (hunk 1 above)
Comment (swarkentin) [reply action.yml:15]: 👆 right, this is basically the same path I went through :)

**Pull request merged**